### ISSUE TITLE
Grant Edits for Specific Ranks in Certain Forum Sections.

### ISSUE THREAD

**Issue by Spikestuff** [adelikat/tasvideos#1889]
Obviously this would have to be set up by the Admin or Moderator, but I was thinking about the [Rejected Submissions](https://tasvideos.org/Forum/Topics/22780) thread and how it primarily requires Judges to go through the list, and "check it off" essentially.

If people become inactive, or don't check the thread often, I believe a fair thing to do is allow the rank that's required (Judge in this case) if given the permission for the forum creation to basically strikethrough the comments on what they've looked over.

**Comment by YoshiRulz** [comment]
Can you think of any other applications for this feature? It sounds like a wiki page (or for moderation tasks, modmail) would be better suited for such lists.

**Comment by Noxxaa** [comment]
I agree with YoshiRulz. This is a square peg/round hole problem - if you need to have multiple users check off a list of movies, use a wiki page for that, or a Google Sheet if you need something more powerful.

This is asking to expand moderation tools for a specific use case that has nothing to do with moderation.
